cosmo_tester/test_suites/service_composition/shared_resource_test.py

```diff
@@ -20,3 +24,59 @@ def test_shared_resource(image_based_manager, ssh_key, logger, test_config):
         _verify_deployments_and_nodes(app, 2)
         _check_custom_execute_operation(app, logger)
         _verify_custom_execution_cancel_and_resume(app, logger)
+
+
+@pytest.fixture(scope='function')
+def managers(request, ssh_key, module_tmpdir, test_config, logger):
+    hosts = Hosts(ssh_key, module_tmpdir, test_config, logger, request, 2)
+    try:
+        hosts.create()
+        yield hosts.instances
+    finally:
+        hosts.destroy()
+
+
+def test_external_shared_resource_idd(managers, ssh_key, logger, test_config):
+    tenant = 'test_external_shared_resource'
+    local_mgr, external_mgr = managers
+
```

Comment:
So if you were to do something with the password, maybe changing the password (there's a util for it in snapshot tests currently I think), then adding a new account and using the new account would be good? That'd ensure it used both the username and password in case we did something really silly in the code in future.

Reply:
(or made a naive mistake like making it always use the current credentials or something)